libs/ledger-live-common/src/families/vechain/js-synchronisation.ts

```diff
@@ -0,0 +1,90 @@
+import type { GetAccountShape } from "../../bridge/jsHelpers";
+import { BigNumber } from "bignumber.js";
+import { makeSync, makeScanAccounts, mergeOps } from "../../bridge/jsHelpers";
+import { encodeAccountId, encodeTokenAccountId } from "../../account";
+import eip55 from "eip55";
+import { emptyHistoryCache } from "../../account";
+
+import { getAccount, getLastBlockHeight, getOperations, getTokenOperations } from "./api";
+import { findTokenById, getTokenById } from "@ledgerhq/cryptoassets/tokens";
+
+const getAccountShape: GetAccountShape = async info => {
+  const { initialAccount, currency, derivationMode } = info;
+  const address = eip55.encode(info.address);
+
+  const oldOperations = initialAccount?.operations || [];
+  const startAt = oldOperations.length ? (oldOperations[0].blockHeight || 0) + 1 : 1;
+
+  const accountId = encodeAccountId({
+    type: "js",
+    version: "2",
+    currencyId: currency.id,
+    xpubOrAddress: address,
+    derivationMode,
+  });
+
+  // get the current account balance state depending your api implementation
+  const { balance, energy } = await getAccount(address);
+
+  // get the current block height
+  const blockHeight = await getLastBlockHeight();
+
+  // Merge new operations with the previously synced ones
+  const newOperations = await getOperations(accountId, address, startAt);
+
+  //Get last token operations
+  const vthoAccountId = encodeTokenAccountId(accountId, findTokenById("vechain/vtho")!);
+  const vthoOperations = await getTokenOperations(
+    vthoAccountId,
+    address,
+    "0x0000000000000000000000000000456e65726779",
```

Comment:
I understand this is VTHO token address, maybe we could store it in a const variable named something like VTHO_TOKEN_ADDRESS

minor

Reply:
done thanks